pkg/network/node/node.go

```diff
@@ -370,9 +365,6 @@ func (node *OsdnNode) Start() error {
 			return err
 		}
 	}
-	if !node.useConnTrack {
-		node.watchServices()
```

Comment:
This makes `watchServices` dead code, which then makes a bunch of other things dead code. See https://github.com/danwinship/sdn/commit/fc0e4b06